woocommerce/woocommerce-gutenberg-products-block
# Update @wordpress/e2e-test-utils-playwright package

## What

Fixes #11865 

## Why

This PR updates the `@wordpress/e2e-test-utils-playwright` package that includes the patch for the spinner and removes the util function we introduced.

It is worth mentioning that now when the `npm install` is executed, this warning is displayed:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'lighthouse@10.4.0',
npm WARN EBADENGINE   required: { node: '>=16.16' },
npm WARN EBADENGINE   current: { node: 'v16.15.0', npm: '8.5.5' }
npm WARN EBADENGINE }
```

I don't see it as a big issue. Also, it is a good reminder to remind that have to update the node version we currently use!  




## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that playwright tests pass.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
